pepper/Makefile

```diff
@@ -43,15 +43,20 @@ link-exo: check-env
 	ln -sf ~/dex-zksnarks/exo_compute/private_input.py $(PEPPER_BIN)/exo0
 
 # Target to run all tests (e.g. used by travis)
-test: unit-tests gadget-tests e2e-tests
+test: unit-tests-iBV unit-tests gadget-tests e2e-tests
```

Comment:
Let's keep this `test` formula small and rather have `unit-tests` be a target that lists all possible unit tests sub-targets (maybe there is a way we can do this in one formula with a loop for each `_test.cpp` file in test/apps/

Reply:
recursive testing all files is done